app/controllers/api/v1/friends_controller.rb

```diff
@@ -66,4 +77,7 @@ def friend_params
       :available_to_party
     )
   end
+  def current_friend
+    @friend =Friend . find(params[:id])
+  end
```

Comment:
I don't think this works. 

You should be doing this within the 'update' method.

The abstraction is not worth the savings of 1 line over the update, show, and delete method.